### DISCUSSION TITLE
Data: Is there a way to update the data by adding +1 to the existing number?

### DISCUSSION THREAD

**Discussion by ru88s** [supabase/supabase#1170] [Questions] [answered]
I want to easily update the number of comments.

![スクリーンショット 2021-04-16 13 59 32](https://user-images.githubusercontent.com/42270511/114973812-2bc05f80-9ebc-11eb-90d4-e1f734e95fa4.png)

**Comment by silentworks** [comment, marked as answer]
You can increment this using a stored procedure. I have answered this before here https://github.com/supabase/supabase/discussions/909#discussioncomment-546117

**Reply by ru88s** [reply]
@silentworks 
Thank you for the amazing technique!
I'll try!

**Reply by ru88s** [reply]
@silentworks 
I’ve done it.
Thank you so much!
